doc/releasenotes/16_0_0_summary.md

```diff
@@ -35,6 +35,24 @@ In [PR #11097](https://github.com/vitessio/vitess/pull/11097) we introduced nati
 
 ### Breaking Changes
 
+#### vtctld UI Removal
+In v13, the vtctld UI was deprecated. As of this release, the `web/vtctld2` directory is deleted and the UI will no longer be included in any Vitess images going forward. All build scripts and the Makefile have been updated to reflect this change.
+
+However, the vtctld HTTP API will remain at `{$vtctld_web_port}/api`.
+
+#### vtctld Flag Deletions
+With the removal of the vtctld UI, the following vtctld flags have been removed:
+- `--vtctld_show_topology_crud`: This was a flag that controlled the display of CRUD topology actions in the vtctld UI. The UI is removed, so thi flag is no longer necessary.
```

Comment:
If we did not officially deprecate this flag in a previous release, we need it to go through a deprecation cycle.